src/main/java/gui/misc/BlurHelper.java

```diff
@@ -1,46 +1,60 @@
 package gui.misc;
 
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.event.EventHandler;
 import javafx.geometry.Bounds;
 import javafx.geometry.Point2D;
 import javafx.geometry.Rectangle2D;
 import javafx.scene.Node;
 import javafx.scene.SnapshotParameters;
+import javafx.scene.control.ScrollPane;
 import javafx.scene.effect.GaussianBlur;
+import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
+import javafx.scene.image.PixelFormat;
 import javafx.scene.image.WritableImage;
+import javafx.scene.input.ScrollEvent;
+import javafx.scene.shape.Rectangle;
 import lombok.Getter;
 import lombok.Setter;
 
 /**
  * Created by markv on 5/3/2016.
  * Class to assist in quickly blurring behind a supplied node.
- * Also has support for moving elements!
+ * Also has support for moving elements.
+ * Is kind of extremely buggy. May have horrible performance impact.
  */
 public class BlurHelper {
 
-    private WritableImage writableImage;    // Image used for writing snapshots to.
-    private SnapshotParameters parameters; // Parameters used for writing snapshots (resolution etc)
-    private Bounds bounds; // Bounds of node used for snapshot parameters (x, y, width, height)
-
+    /**
```

Comment:
Javadoc usually belongs to a method or specific variable. This comment is about multiple variable, so make it a general line or block comment.